src/request/src/middleware/src/storage/index.js

```diff
@@ -66,7 +66,7 @@ export default class Storage {
 
   findById(collection, id) {
     if (isString(id) === false) {
-      return Promise.reject(new Error('id must be a string', id));
+      return Promise.reject(new KinveyError('id must be a string', id));
```

Comment:
The `debug` property is used to provide extra helpful information on how to resolve or fix and error that may have occurred. It was really a carry over from version 1.x of the JS SDK and isn't as widely used in 3.x.